packages/undbx/PKGBUILD

```
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=undbx
+pkgver=undbx.0.21.r3.g5e31c75
+pkgrel=1
+pkgdesc='Extract e-mail messages from Outlook Express DBX files.'
+arch=('any')
+groups=('blackarch' 'blackarch-forensic')
+url='https://github.com/ZungBang/undbx'
+license=('GPL')
+depends=()
+makedepends=('autoconf' 'automake' 'git')
```

```suggestion
makedepends=('git')
```

cf. https://github.com/BlackArch/blackarch/pull/2619